src/applications/common/schemaform/save-in-progress/FormStartControls.jsx

```diff
@@ -90,21 +111,37 @@ class FormStartControls extends React.Component {
   }
 }
 
+
 FormStartControls.propTypes = {
+  prestartForm: PropTypes.func,
+  form: PropTypes.object.isRequired,
   formId: PropTypes.string.isRequired,
   handleLoadPrefill: PropTypes.func,
   migrations: PropTypes.array,
   returnUrl: PropTypes.string,
   fetchInProgressForm: PropTypes.func.isRequired,
   removeInProgressForm: PropTypes.func.isRequired,
-  router: PropTypes.object.isRequired,
   formSaved: PropTypes.bool.isRequired,
   prefillAvailable: PropTypes.bool.isRequired,
   startPage: PropTypes.string.isRequired,
   startText: PropTypes.string,
   resumeOnly: PropTypes.bool
 };
 
-export default withRouter(FormStartControls);
+function mapStateToProps(state) {
+  return {
+    form: state.form
```

Comment:
Good question @markgreenburg, in this case, the `form` actually refers to the `formConfig`, which is a required parameter for `beforeStartForm` (which has been renamed `prestartCheck`). Should I set up an issue to discuss renaming this prop so to avoid this kind of confusion in the future?

Reply:
I have also renamed it to `formConfig` in mapStateToProps in an effort to improve the clarity of this for now.

Reply:
Actually it looks like we can remove the formConfig as an argument to prestartForm, as that is likely a premature optimization (to allow for reuse of shared prestartChecks between different forms, such as is likely to occur on these disability forms). Good catch @markgreenburg. Will update.